.github/workflows/build-test-javascript.yaml

```diff
@@ -4,7 +4,17 @@ name: build-test-javascript
 
 on:
   workflow_dispatch:
+    inputs:
+      upload-artifacts:
```

Comment:
I think we usually use 'dry-run' in the other workflows when we don't do the actual modification, but I'm fine with upload-artifacts.